app/Http/Controllers/AdminDash.php

```diff
@@ -471,6 +471,13 @@ public function disallowVisitReq($id) {
     public function allowVisitReq(Request $request) {
         $id = $request->cid;
         $visitrej = VisitRej::where('cid', $id)->first();
+
+        $validator = $request->validate([
+            'cid' => 'required'
+        ]);
```

Comment:
Let's move this up to the first line in the function (line 472). We should run the validation before trying to use the input

Reply:
Good point. I've just updated it with a new commit.